WebdevPeriod3/Controllers/DashboardController.cs

```diff
@@ -78,12 +73,12 @@ public async Task<IActionResult> Profile()
         {
             
             // RETRIEVE USER INFORMATION
-            User user = await _userManager.GetUserAsync(User);
+            var user = await _userManager.GetUserAsync(User);
```

Comment:
Why did you change this with var?

Reply:
I wrote it to begin with. No reason to type out User anyways when var is available